ledger/eval.go

```diff
@@ -394,11 +389,6 @@ func (eval *BlockEvaluator) TestTransactionGroup(txgroup []transactions.SignedTx
 // on a single transaction, but does not actually add the transaction to the block
 // evaluator, or modify the block evaluator state in any other visible way.
 func (eval *BlockEvaluator) testTransaction(txn transactions.SignedTxn, cow *roundCowState) error {
-	// Verify that groups are supported.
```

Comment:
What's the reason that we're dropping this check here?

Reply:
it's redundant with a check inside WellFormed()

Reply:
Ah, for some reason I missed that.